src/test/java/com/ionosenterprise/rest/test/BackupUnitTest.java

```diff
@@ -0,0 +1,87 @@
+package com.ionosenterprise.rest.test;
+
+import com.ionosenterprise.rest.client.RestClientException;
+import com.ionosenterprise.rest.domain.BackupUnit;
+import com.ionosenterprise.rest.domain.BackupUnits;
+import com.ionosenterprise.rest.domain.SingleSignOnUrl;
+import com.ionosenterprise.rest.test.resource.BackupUnitResource;
+import com.ionosenterprise.rest.test.resource.CommonResource;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+
+import static org.junit.Assert.*;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class BackupUnitTest extends BaseTest {
+
+    private static String backupUnitId;
+
+    @BeforeClass
+    public static void t1_testCreateBackupUnit() throws NoSuchMethodException, IOException, IllegalAccessException,
+            RestClientException, InvocationTargetException, InterruptedException {
+
```

Comment:
remove empty line

Reply:
I see this empty line necessary in case of indented form new line list of thrown errors to be cleaner where the method signature ends and actual method code starts.